pypeit/inputfiles.py

```diff
@@ -935,22 +935,29 @@ def options(self):
 
         # Load coordinate offsets for each file. This is "Delta RA cos(dec)" and "Delta Dec"
         # Get the RA offset of each file
-        off_ra = self.path_and_files('ra_offset', skip_blank=False, check_exists=False)
-        if off_ra is None:
-            opts['ra_offset'] = None
-        elif len(off_ra) == 1 and len(self.filenames) > 1:
-            opts['ra_offset'] = off_ra*len(self.filenames)
-        elif len(off_ra) != 0:
-            opts['ra_offset'] = off_ra
+        off_ra, off_dec = None, None
+        if 'ra_offset' in self.data.keys():
+            off_ra = self.data['ra_offset'].tolist()
+            if off_ra is None:
```

Comment:
Is this ever true?  Or should you first check if `off_ra` is None before doing the list conversion?